tests/vm_test.go

```diff
@@ -303,32 +288,6 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			Expect(vmi.Annotations).To(HaveKeyWithValue("testannotation", "test"))
 		})
 
-		It("[test_id:3162]should ignore kubernetes and kubevirt annotations to VMI", func() {
```

Comment:
Maybe this check can be moved to above test to cover the behavior as whole

Reply:
Isn't what  you wanted to check with the new unit test?

Reply:
Yes, that is true and covering this by both unit and e2e wouldn't hurt us. If we move this check to the existing test it will not increase the runtime.

Reply:
Which existing test? the above `[test_id:3161]should carry annotations to VMI` test is setting the annotations to the VMI on creation and not to the VM.

Reply:
@xpivarc I think the unit tests coverage is enough here since this is sort of a negative test in the sense that it covers up something undesired we had introduced a while ago accidentally

Reply:
The `[test_id:3161]should carry annotations to VMI` can just do the negative assert as well. This is user facing behavior so having an e2e test for it is nice.

Reply:
Thanks @xpivarc. I think unit test should cover it, we'd like respect to not over-testing it.